HACKING.md

```diff
@@ -141,17 +141,82 @@ appropriately.  To install `godep` locally run:
 
 If you are not updating packages you should not need godep installed.
 
-## Updating Godeps from upstream
+## Updating Kubernetes from upstream
+
+There are a few steps involved in rebasing Origin to a new version of Kubernetes. We need to make sure
```

Comment:
Yeah - maybe the script should show the diff first, and say "hey are you ok with all of these"?  It's easy to skip that step (I have as well).  The UPSTREAM: commits might be resetting those versions as well.